config.js

```diff
@@ -45,7 +45,9 @@ module.exports = {
       },
       apply: ['establishment:*'],
       update: ['establishment:admin', 'project:own'],
-      updateConditions: ['asru:*']
+      updateConditions: ['asru:*'],
+      delete: ['establishment:admin', 'project:own'],
+      revoke: ['asru:*', 'establishment:admin', 'establishment:read', 'project:own']
```

Comment:
Pretty sure you don't want `establishment.read` in here.
```suggestion
      revoke: ['asru:*', 'establishment:admin', 'project:own']
```